test/scenarios.py

```diff
@@ -5773,6 +5847,56 @@
                     action.CancelTransferRequest(0),
```

Comment:
The receiver should also receive Paused event if the sender did the `stop()` and vice versa

Reply:
It does https://github.com/NordSecurity/libdrop/pull/171/files/614465fcdf351d7d06de0b936214015313ea4d73#diff-01281065a74eabdb084c366fb316dda3f5defaa50ed1f54df6bcac6209f35297L5759